frontend/shared/src/components/stepper/Stepper.tsx

```diff
@@ -18,10 +18,8 @@ const Stepper: React.FC<StepperProps> = ({ activeStep = 1, steps }) => (
   </$StepsContainer>
 );
 
-const defaultProps = {
+Stepper.defaultProps = {
   activeStep: 1,
 };
 
-Stepper.defaultProps = defaultProps;
-
```

Comment:
I think we should get rid of these `defaultProps`, this `({ activeStep = 1, steps }) => (` would do the trick.

Reply:
Let's try if I can remove it. I think that eslint complained about when not having it.

Reply:
seem to work!